ConfigSpace/forbidden.py

```diff
@@ -72,12 +74,13 @@ def is_forbidden(self, instantiated_hyperparameters):
 
 
 class AbstractForbiddenClause(AbstractForbiddenComponent):
-    def get_descendant_literal_clauses(self):
+    def get_descendant_literal_clauses(self) -> List['AbstractForbiddenClause']:
```

Comment:
changed. 
also its different class so no need to declare it as str.
why string? http://stackoverflow.com/questions/33533148/how-do-i-specify-that-the-return-type-of-a-method-is-the-same-as-the-class-itsel